marker/v2/processors/sectionheader.py

```diff
@@ -26,7 +26,7 @@ def __call__(self, document: Document):
                 if block.block_type not in self.block_types:
                     continue
 
-                line_heights[block.block_id] = [document.get_block(l).polygon.height for l in block.structure if l.block_type == BlockTypes.Line]
+                line_heights[block.block_id] = [document.get_block(l).polygon.height for l in block.structure if block.structure is not None and l.block_type == BlockTypes.Line]
```

Comment:
A better way to do this is to assign an empty list to line heights if block.structure is None, I think.  I think it won't fix the error now, since it will still try to iterate block.structure

Reply:
You're right, forgot to update this file!